doc/examples/edges/plot_circular_elliptical_hough_transform.py

```diff
@@ -82,10 +82,10 @@
 # -------------------
 #
 # The algorithm takes two different points belonging to the ellipse. It
-# assumes that it is the main axis. A loop on all the other points determines
-# how much an ellipse passes to them. A good match corresponds to high
-# accumulator values.
-#
+# assumes that it is the major axis. A loop on all the other points determines
+# potential minor axis lengths of candidate ellipses. Ellipses are added to the
+# results if enough valid candidates have similar minor axis lengths.
```

Comment:
Thank you so much for simplifying the explanation, @adfoucart. Here, I would be tempted to add a parenthesis about 'valid' though. Something like "(by valid, we are referring to the minor and major axis lengths falling within the prescribed bounds)." @stefanv would you agree?